features/bootstrap/FeatureContext.php

```diff
@@ -14,7 +14,15 @@
 
 define( 'EE', true );
 define( 'EE_VERSION', trim( file_get_contents( EE_ROOT . '/VERSION' ) ) );
-define( 'EE_ROOT_DIR', '/opt/easyengine' );
+
+$root_alias = '/opt';
+if ( 'Darwin' === php_uname( 's' ) ) {
```

Comment:
Use the same code as in init-ee.php for consistency.
https://github.com/EasyEngine/easyengine/blob/34be8fc1d7f7f9f260d730c349dc54a92807f0bc/php/init-ee.php#L7-L13